tqdm/std.py

```diff
@@ -749,11 +750,18 @@ def wrapper(*args, **kwargs):
                     # on the first column/row to decide whether it can
                     # take a fast or slow code path; so stop when t.total==t.n
                     t.update(n=1 if not t.total or t.n < t.total else 0)
-                    return func(*args, **kwargs)
+                    if isinstance(func, dict) or isinstance(func, list):
+                        return func
+                    else:
+                        return func(*args, **kwargs)
 
                 # Apply the provided function (in **kwargs)
                 # on the df using our wrapper (which provides bar updating)
-                result = getattr(df, df_function)(wrapper, **kwargs)
+                if isBuiltinFunction or df_function != "aggregate":
+                    result = getattr(df, df_function)(wrapper, **kwargs)
+                else:
+                    result = getattr(df, df_function)(wrapper, **kwargs)
+                    result = getattr(df, df_function)(func)
```

Comment:
this can't be right - `result` twice...

Reply:
Also `getattr(df, df_function)(func)` won't get any progress.

Reply:
Also getattr(df, df_function)(func) won't get any progress. 
Yes, and that is the reason I used result twice. The wrapper if called would result in a dataframe of the list of functions being assigned as a string to all rows. Thus, I used both, the second result provides for the correct result whereas the first one accounts for the progress. I will try a workaround for the same.

Reply:
@casperdcl Please let me know if the solution works for you

Reply:
Understood but it's not good. I presume `getattr(df, df_function)(wrapper, **kwargs)` could take quite a long time to run on big dataframes - potentially even halving the overall speed?

Reply:
Let me try it on a big database? 
If there is not much difference in the speed I think it would be fine. 
This would allow us to solve this without major changes in the implementation of aggregate as a whole. Anyhow, I will try that too.